### ISSUE TITLE
Load ROM via command line missing in documentation

### ISSUE THREAD

**Issue by tommcc** [tursilion/classic99#3]
My goal is to be able to launch arbitrary TI-99 games from my game launcher. Though using app mode is one way to launch directly to a game, it requires a lot of work when considering many games (copying the .exe, custom .ini, hand-coding the filesizes, etc.).

It would be great to allow Classic99 to load a cart simply by providing a path to the first file as an argument when launching. I'd imagine the same logic for finding multiple "parts" using the filename ("part 1 of n") when adding user carts via UI could apply.

Example:

```
classic99.exe "path\to\Cart (Part 1 of 2).G"
```

The above would allow dragging a cart file onto the .exe file in Windows. However, if command line options were expanded I could see requiring a prefix, maybe even integrating the auto-select functionality from app mode.

If there already is a way to do this, I wasn't able to find it in the docs. Thanks!

**Comment by tursilion** [comment]
Yeah, seems like it's not in the manual. Command line is: classic99.exe 
-rom "path\to\rom.bin"

Rules for ROM identification are the same as using Cartridge->User->Open 
(xxxxxC.BIN, for instance, Classic99 will automatically seek out all 
like-named files and load them in).


Tom McCaffery wrote on 1/6/2020 8:07 AM:

**Comment by tommcc** [comment]
That's great, thanks for the info! Updating issue title to refer to the documentation omission.

**Comment by tursilion** [comment]
oh.. this wasn't an email. I keep forgetting about this thing ;)
